src/ssz/type/list.zig

```diff
@@ -68,6 +70,20 @@ pub fn FixedListType(comptime ST: type, comptime _limit: comptime_int) type {
             mixInLength(value.items.len, out);
         }
 
+        /// Clones the underlying `ArrayList`.
+        ///
+        /// Caller owns the memory.
+        pub fn clone(allocator: std.mem.Allocator, value: *const Type, out: *Type) !void {
+            try out.resize(allocator, value.items.len);
+
+            for (value.items, 0..) |v, i| {
+                var e: Element.Type = undefined;
+                try Element.clone(&v, &e);
+
+                out.items[i] = e;
```

Comment:
is this needed?
can it be `try Element.clone(&v, &out.items[i]);`

Reply:
done in [0107068](https://github.com/ChainSafe/ssz-z/pull/51/commits/01070689f535d7fa2708259e0bd7ab4cc386d285)